go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	k8s.io/code-generator v0.26.3
 	k8s.io/component-base v0.26.3
 	k8s.io/klog/v2 v2.90.1
-	open-cluster-management.io/addon-framework v0.6.1
+	open-cluster-management.io/addon-framework v0.1.0
```

Comment:
@skitt Any idea why this change? I used the `update-submariner.sh` script.

Reply:
No, I haven’t figured out why `go get` downgrades that dependency... You need to manually fix it.